app/Activity.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Activity extends Model
+{
+    /**
+     * Mass assignable fields of Activity model
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'course_id',
+        'name',
+        'grade',
+        'percentage'
```

Comment:
Add a comma at the end of this line: 'percentage',

Itll make revisions easier